# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1257951663 18000
#      Wed Nov 11 10:01:03 2009 -0500
# Node ID 13868ea67c7188721c1bd7e186d0ec0277ed09be
# Parent  ef4c4186cb477cce509827e1ab41f2e14768b89c
configure.ac: fail if LT_INIT is not defined

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2009-11-11  John W. Eaton  <jwe@octave.org>
+
+	* configure.ac: Fail if LT_INIT it is not defined.
+
 2009-11-10  John W. Eaton  <jwe@octave.org>
 
 	* NEWS: Update.
 
 	* configure.ac: Exit with error if BLAS or LAPACK libraries are
 	not found or if the BLAS library is found to be incompatible with
 	the Fortran compiler.  Eliminate warn_blas_f77_incompatible
 	variable.
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -874,16 +874,21 @@ fi
 
 AC_SUBST(GRAPHICS_CFLAGS)
 AC_SUBST(GRAPHICS_LIBS)
 
 # ----------------------------------------------------------------------
 
 OCTAVE_PROG_AR
 
+ifdef([LT_INIT], [], [
+  errprint([error: you must have libtool 2.2.2 or a more recent version
+])
+  m4exit([1])])
+
 LT_PREREQ([2.2.2])
 LT_INIT([disable-static dlopen win32-dll])
 
 if test x$enable_shared = xyes; then
   SHARED_LIBS=true
 else
   SHARED_LIBS=false
 fi
